docker-compose-3-personal.yml

```diff
@@ -0,0 +1,351 @@
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=dscp-identity
+    networks: ['regulator']
+
+  regulator-identity:
+    image: digicatapult/dscp-identity-service:latest
+    container_name: regulator-identity
+    command: /bin/sh -c "
+      sleep 10 &&
+      npx knex migrate:latest &&
+      node app/index.js"
+    ports:
+      - 8021:8021
+    depends_on:
+      - regulator-node
+      - regulator-postgres-identity
+    environment:
+      - PORT=8021
+      - API_HOST=regulator-node
+      - API_PORT=9944
+      - DB_HOST=regulator-postgres-identity
+      - DB_PORT=5432
+      - DB_NAME=dscp-identity
+      - DB_USERNAME=postgres
+      - DB_PASSWORD=postgres
+      - SELF_ADDRESS=5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y
+      - AUTH_TYPE=${AUTH_TYPE:-NONE}
+    networks: ['regulator']
+  regulator-node:
+    image: digicatapult/dscp-node:latest
+    container_name: regulator-node
+    command: --base-path /data/
+      --charlie
+      --unsafe-ws-external
+      --unsafe-rpc-external
+      --rpc-cors all
+      --node-key 0000000000000000000000000000000000000000000000000000000000000003
+      --bootnodes /dns4/alice/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
+    ports:
+      - 32333:30333
+      - 10144:9944
+      - 10133:9933
+    restart: on-failure
+    networks: ['regulator', 'chain']
+
+  regulator-hyproof-api:
+    container_name: regulator-hyproof-api
+    build:
+      context: './'
+      dockerfile: 'Dockerfile'
+    command: /bin/sh -c "
+      sleep 10 &&
+      npx knex migrate:latest --knexfile build/lib/db/knexfile &&
+      npm start"
+    environment:
+      - PORT=8020
+      - NODE_HOST=regulator-node
+      - NODE_PORT=9944
+      - DB_HOST=regulator-postgres-hyproof-api
+      - DB_PORT=5432
+      - DB_NAME=dscp-hyproof-api
+      - DB_USERNAME=postgres
+      - DB_PASSWORD=postgres
+      - IDENTITY_SERVICE_HOST=regulator-identity
+      - IDENTITY_SERVICE_PORT=8021
+      - IPFS_HOST=ipfs
+      - IPFS_PORT=5001
+      - USER_URI=//Charlie
+    ports:
+      - 8020:8020
+    depends_on:
+      - regulator-node
+      - regulator-identity
+      - regulator-postgres-hyproof-api
+      - ipfs
+    restart: on-failure
+    networks: ['regulator', 'ipfs']
+
+  # ------------------------------------------------------------------------ ipfs
+  ipfs:
+    image: ipfs/go-ipfs:v0.23.0
+    container_name: ipfs
+    environment:
+      - |
+        IPFS_SWARM_KEY=/key/swarm/psk/1.0.0/
+        /base16/
+        0000000000000000000000000000000000000000000000000000000000000000
+    ports:
+      - 4001:4001
+      - 8080:8080
+      - 5001:5001
+    networks: ['ipfs']
+    volumes:
+      - ipfs:/data/ipfs
+
+volumes:
+  energy-owner-hyproof-api-storage:
+  energy-owner-identity-storage:
+  hydrogen-owner-hyproof-api-storage:
+  hydrogen-owner-identity-storage:
+  regulator-hyproof-api-storage:
+  regulator-identity-storage:
+  ipfs:
```

Comment:
This is not that important .. ( the GitHub files changed diff is showing me this red circle at the end of the file which indicates `No newline at end of file` is there a best practice to add that newline character ? I don't know :( just a thought ..